tests/tests/test_zookeeper.py

```diff
@@ -1,7 +1,7 @@
-import kubectl
-import clickhouse
-import settings
-import util
+import tests.clickhouse as clickhouse
```

Comment:
I ask again, why we rename `tests` to `tests/tests` it look so ugly

Reply:
So that root project folder don't contain a bunch of tests-related files, such as `regression.py`, `image` folder, etc